tests/test_study.py

```diff
@@ -454,10 +455,14 @@ def test_study_trials_dataframe_with_no_trials():
 
 
 @pytest.mark.parametrize('storage_mode', STORAGE_MODES)
-@pytest.mark.parametrize('include_internal_fields', [True, False])
+@pytest.mark.parametrize('attrs', [
+    ('number', 'value', 'datetime_start', 'datetime_complete', 'params', 'user_attrs',
+     'system_attrs', 'state'),
+    ('number', 'value', 'datetime_start', 'datetime_complete', 'params', 'user_attrs',
+     'system_attrs', 'state', 'intermediate_values', '_trial_id', 'distributions')])
 @pytest.mark.parametrize('multi_index', [True, False])
-def test_trials_dataframe(storage_mode, include_internal_fields, multi_index):
-    # type: (str, bool, bool) -> None
+def test_trials_dataframe(storage_mode, attrs, multi_index):
+    # type: (str, Tuple[str], bool) -> None
```

Comment:
Should it be `Tuple[str, ...]` ?

Reply:
You're absolutely right. Fixed: https://github.com/optuna/optuna/pull/775/commits/8ac3072cbe5fdefe542e4a34aeaee3b166eacd9f.